src/components/RecoveryPhraseInput.tsx

```diff
@@ -68,7 +68,7 @@ export default function RecoveryPhraseInput({
   return (
     <Card
       rounded={true}
-      shadow={showInput ? Shadow.SoftLight : null}
+      shadow={showInput ? null : Shadow.SoftLight}
```

Comment:
@finnian0826 It looks like `showInput` is always returning true, but this looks to be a preexisting bug.

Reply:
I think showInput returns false when you click the restore button and it is processing. Talked with product and they said the style should change then rather than when a full phrase is entered, so style change might only be for a brief moment if the processing is quick

Reply:
https://github.com/valora-inc/wallet/assets/140328381/391b9676-33a6-42e3-bea2-518276ebde56